dashboard/app/models/script_level.rb

```diff
@@ -37,6 +36,22 @@ def script
     Script.get_from_cache(script_id)
   end
 
+  def level
+    levels[0]
+  end
+
+  def level=(l)
+    levels[0] = l
+  end
+
+  def level_id
+    levels[0].id
+  end
+
+  def level_id=(new_level_id)
+    levels[0] = Level.find(new_level_id)
+  end
```

Comment:
I'm not sure how I feel about these convenience methods. I think they might be doing a little _too_ much to disguise the fact that ScriptLevel can now have many levels. How necessary are these, and how difficult would it be to rename them to use something like `first_level` rather than just `level`?